packages/account/src/Sections/Profile/PhoneVerification/confirm-phone-number.tsx

```diff
@@ -127,51 +187,77 @@ const ConfirmPhoneNumber = observer(({ show_confirm_phone_number, setOtpVerifica
                     'phone-verification__card--inputfield--error': error_message,
                 })}
             >
-                <TextFieldAddon
-                    type='number'
-                    label={localize('Phone number')}
-                    value={phone_number}
-                    status={error_message ? 'error' : 'neutral'}
-                    message={error_message}
-                    className='phone-verification__card--inputfield__phone-number-input'
-                    onChange={handleOnChangePhoneNumber}
-                    addonLabel='+'
-                />
+                {isCountryCodeDropdownEnabled ? (
+                    formatted_countries_list &&
+                    !isLoading && (
```

Comment:
what if isLoading is true?

Reply:
it will not show the country code drop down until it's being loaded